controllers/userController.js

```diff
@@ -55,23 +55,18 @@ module.exports = {
       }
 
       const passwordCompare = await bcrypt.compare(password, user.password);
-      // console.log(passwordCompare);
       if (passwordCompare) {
-        const { id: user_id, username: foundedUsername } = user;
-
         let token = crypto
           .createHash("md5")
-          .update(foundedUsername)
+          .update(`${username}${password}${process.env.SECRETKEY}`)
           .digest("hex");
-
-        const exTime = new Date();
-
-        await Access_Token.create({
-          user_id,
-          access_token: token,
-          expiry: exTime,
+        const expiryTime = new Date();
+        const createToken = await Access_Token.create({
```

Comment:
what happens, is if a user logs in 5 min ago, and tries to log in again, this function will always give an error because the user already has a token stored. and userId is unique so SQL will stop you from creating new token. 
you need to tackle this issue as well.

Reply:
resolved.